base_usability/__manifest__.py

```diff
@@ -40,7 +40,7 @@
     ],
     'test': [
```

Comment:
@jjscarafia I just saw this test key in the module descriptor.
Should I remove it?